storage/local/crashrecovery.go

```diff
@@ -42,6 +42,12 @@ func (p *persistence) recoverFromCrash(fingerprintToSeries map[clientmodel.Finge
 	count := 0
 	seriesDirNameFmt := fmt.Sprintf("%%0%dx", seriesDirNameLen)
 
+	// Delete the fingerprint mapping file as it might be stale or
+	// corrupt. We'll rebuild the mappings as we go.
```

Comment:
Let's say I have two series with colliding fingerprints. Don't their exact mapped fingerprint values depend on the order of their first appearance? And if we reconstruct the mapped fingerprints from the indexes, we could get the reverse order and the fingerprints are mapped incorrectly?